jbmc/regression/jbmc-strings/ConstantEvaluationEqualsIgnoreCase/pi.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+Main.class
+--function Main.pi --cp `../../../../scripts/format_classpath.sh . ../../../lib/java-models-library/target/core-models.jar`
+^Generated [1-9]\d* VCC\(s\), [1-9]\d* remaining after simplification$
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+--
+--
```

Comment:
Might want to note *why* this one doesn't work, either here or in a comment in the java file